vendor/bat-native-ads/src/bat/ads/internal/processors/behavioral/bandits/epsilon_greedy_bandit_segments.cc

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/behavioral/bandits/epsilon_greedy_bandit_segments.h"
+
+namespace {
+
+constexpr const char* kSegments[] = {"architecture",
```

Comment:
maybe convert this to `base::StringPiece[]` as other arrays?

Reply:
yes, makes sense